.github/workflows/test.yml

```diff
@@ -12,17 +12,26 @@ concurrency:
   group: ${{ github.workflow }}-${{ github.ref }}
   cancel-in-progress: true
 
+
+env:
+  FORCE_COLOR: '1' # Make tools pretty.
+  TOX_TESTENV_PASSENV: FORCE_COLOR
+  PIP_DISABLE_PIP_VERSION_CHECK: '1'
+  PIP_NO_PYTHON_VERSION_WARNING: '1'
+  PYTHON_LATEST: '3.12'
+
+
 jobs:
   lint:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v3
+      - uses: actions/checkout@v4
         with:
           fetch-depth: 0
-      - name: Set up Python 3.8
-        uses: actions/setup-python@v4
+      - uses: actions/setup-python@v4
         with:
-          python-version: '3.8'
+          python-version: ${{ env.PYTHON_LATEST }}
+      - uses: pre-commit/action@v3.0.0
       - uses: pre-commit/action@v2.0.3
```

Comment:
```suggestion
```

as it is extra now

Reply:
Thanks! I'll have to also remove the `token` and the ability to push the changes back to the PR since it doesn't work on version 3.0.0 anymore